httpx/utils.py

```diff
@@ -169,3 +169,7 @@ def get_logger(name: str) -> logging.Logger:
             logger.addHandler(handler)
 
     return logging.getLogger(name)
+
+
+def to_bytes(value: typing.Union[str, bytes]) -> bytes:
```

Comment:
We should take an encoding parameter here too, default to UTF-8.

Reply:
Fixed!